website/src/hoc/index.js

```diff
@@ -1,6 +1,7 @@
 import withFirebase from './firebase/firebase';
 import WithProtectedView from './WithProtectedView/WithProtectedView';
+import WithFirebaseSimple from './firebase/WithFirebaseSimple';
 
 export {
-  withFirebase, WithProtectedView,
+  withFirebase, WithProtectedView, WithFirebaseSimple,
```

Comment:
same as above, don't need `WithFirebaseSimple` i think.

Reply:
`WithFIrebaseSimple` main purpose is to make it easier to test components that rely on Firebase for their data